react/training/src/redux/game/reducers.js

```diff
@@ -0,0 +1,45 @@
+import { actionTypes } from './actions';
+
+const initialState = {
+  history: [{ squares: Array(9).fill(null) }],
+  stepNumber: 0,
+  xIsNext: true,
+  winner: null
```

Comment:
`winner: ''`. Try to avoid `null` or  `undefined` values. In this case we can use `''`